### PR TITLE
[P2022-215] routing rebuilding

### PR DESCRIPTION
**Pull request by jprymak**

### CODE REVIEW

File: src/links/links.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+interface objectType {
```
Comment (konradkierus) [inline src/links/links.ts:1]: interfaces should start with capital letter

File: src/links/links.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  id: number;
+  url: string;
+  text: string;
+}
+
+export const links: objectType[] = [
```
Comment (konradkierus) [inline src/links/links.ts:7]: please convert to enum, e.g.
```
enum Pages {
  Home: '/',
  Secondary: '/second'
}
```

**Review by konradkierus — CHANGES_REQUESTED** [review]

**Comment by konradkierus** [comment]
Please also use [P2022-215] PR naming convention when you put ticket number in PR title.

File: src/links/links.ts (hunk 1 above)
Comment (jprymak) [reply src/links/links.ts]: thanks :)

File: src/links/links.ts (hunk 2 above)
Comment (jprymak) [reply src/links/links.ts]: i suppose when defining enums assignment operators should be used instead of ":"

Comment (konradkierus) [reply src/links/links.ts]: @jprymak yes, sorry, I mixed the syntax when I was writing my comment :)

**Review by konradkierus — APPROVED** [review]

**Pull request merged**